ansys/dpf/core/custom_fields_container.py

```diff
@@ -232,15 +282,25 @@ def get_fields_by_mat_id(self, matid, timeid=None, complexid=None):
         return self.get_fields(label_space)
     
     def get_field_by_mat_id(self, matid, timeid=None, complexid=None):
-        """Returns a a field with the given material id.
-        If a timeid or a complexid (0 for real and 1 for imaginary) are given, 
-        the field returned is for a given time and/or 
-        complex type. The available mat id can be requested with `get_mat_scoping`
+        """Retrieve a field with a given material ID.
         
+         You can filter the field based on a given time, complex type, or both. 
+                
+        Parameters
+        ----------
+        matid : int, optional
+           Material ID. To request available material IDs, you can use 
+           the :func:`get_mat_scoping` function.
```

Comment:
You'll need to add in the class since this is a method and not a function.  If you can't find it, it's ok, we can add it in later.

Reply:
This one i didn't link. The last method was 'get_mat_scoping` function. I'll check the rendered output.